epochStart/metachain/systemSCs.go

```diff
@@ -313,6 +317,13 @@ func (s *systemSCProcessor) ProcessSystemSmartContract(
 		}
 	}
 
+	if s.flagStakingV4Enabled.IsSet() {
```

Comment:
rename to flag Init

Reply:
287 you need a disable flag

Reply:
- Renamed to `flagStakingV4InitEnabled`
- Also added a disable flag